### ISSUE TITLE
CA1041 doesn't fire in VS2015 but does in VS2013

### ISSUE THREAD

**Issue by hvanbakel** [dotnet/roslyn-analyzers#249]
Given code like this:
```CSharp
[Obsolete]
public void Test() { } 
```

VS2015 will not throw a violation when it certainly should.

**Comment by genlu** [comment]
@hvanbakel Hi, did I miss something? This seems like a bug in FxCop for VS2015, not Roslyn analyzers.

**Comment by hvanbakel** [comment]
That's correct, you're not missing anything. I asked in the Roslyn gitter
where to put this question since I don't know where to log fxcop errors.

**Comment by genlu** [comment]
@hvanbakel Gotcha. I will let FxCop team know. Thank you for reporting this.

@srivatsn @mavasani

**Comment by tmeschter** [comment]
@srivatsn Where would you like this bug?

**Comment by srivatsn** [comment]
We are unlikely to fix existing FxCop rule implementations since we are going to be investing in Rosyn-based implementations for them going forward. We should turn this into an issue about implementing CA1041 using Roslyn. Since the rule is about ObsoleteAttribute which is in System.Runtime.dll, I would add this analyzer to System.Runtime.Analyzers.dll

**Comment by YinzcamJoseph** [comment]
:+1:

**Comment by genlu** [comment]
I have created an issue (https://github.com/dotnet/roslyn-analyzers/issues/252) per @srivatsn's comment.
